src/uptane/secondaryconfig.h

```diff
@@ -23,13 +23,12 @@ class SecondaryConfig {
   std::string ecu_serial;
   std::string ecu_hardware_id;
   bool partial_verifying;
-  std::string ecu_private_key_;
-  std::string ecu_public_key_;
+  std::string ecu_private_key;
+  std::string ecu_public_key;
```

Comment:
Looks good and I'm going to go ahead and merge this in, but the original reason for the duplicated `ecu_*_key_` members and `ecu_*_key` functions was to have functions that would automatically handle the case where the complete path was included without needing to use `full_client_dir`. This came up in one of Robert's tests. Not a big deal for now, but I like having one place where we normalize paths.

Reply:
Yes, makes sense. I'd prefer to get rid of full_client_dir at all, it only makes stuff more complicated. I hope backwards compatibility is not an issue yet.

Reply:
I won't argue with that. Anything to make it simpler.

I manually rebased this into the main branch, but since there were (trivial) conflicts with my branch, github wasn't smart enough to automatically recognize what I did.